Nos/Views/Moderation/UserFlagView.swift

```diff
@@ -0,0 +1,137 @@
+import SwiftUI
+
+/// Displays pickers for selecting user flag options, with additional stages shown
+/// based on previous selections.
+struct UserFlagView: View {
+    @Binding var selectedFlagOptionCategory: FlagOption?
+    @Binding var selectedSendOptionCategory: FlagOption?
+    @Binding var selectedVisibilityOptionCategory: FlagOption?
```

Comment:
We could probably remove the word Category from these.

Reply:
Thanks for pointing this out!